Gododir/gen.go

```diff
@@ -216,24 +239,36 @@ var (
 			},
 		},
 	}
+	mutex = &sync.Mutex{}
```

Comment:
Let's rename this to be a little more specific. Maybe `fails_lock`? I'm relatively certain I've seen somewhere that there is a standard idiom for this exact sorta scenario, but I can't remember exactly what it was. :/

Reply:
Fixed